sound/soc/intel/boards/bytcr_rt5651.c

```diff
@@ -1127,6 +1127,7 @@ static int snd_byt_rt5651_mc_probe(struct platform_device *pdev)
 	return ret_val;
 
 err:
+	device_remove_properties(priv->codec_dev);
```

Comment:
This will also get called when bytcr_rt56??_add_codec_device_props() fails since the error handling for that has a goto err;. You probably need to avoid a new err_foo label to avoid this.

Reply:
yes, nice catch, fixed now